set.mm

```diff
@@ -659240,8 +659175,7 @@ unification theorem (e.g., the sub-theorem whose assertion is step 5
     $( A singleton index picks out an instance of an indexed union's argument.
        (Contributed by Glauco Siliprandi, 17-Aug-2020.) $)
     iunxsngf2 $p |- ( A e. V -> U_ x e. { A } B = C ) $=
```

Comment:
This theorem can be removed!?

Reply:
Probably yes, given the current proof (unless @glacode says differently).

Reply:
commit d6a37f4b30c1b0e4db3b0d382a10c8578cf2e2b0
moved iunxsngf from @tirix  mathbox to main, on Fri Oct 14 2022, I'm not entirely sure how to reconcile this with the iunxsngf's comment (Contributed by Mario Carneiro, 25-Jun-2016.)

In any case, I'm okay with its removal. Kindly ensure to update the proofs that make reference to iunxsngf  (iunxsnf , if a minimize didn't already change such reference)

Reply:
> I'm not entirely sure how to reconcile this with the iunxsngf's comment (Contributed by Mario Carneiro, 25-Jun-2016.) 

Mario is the original author of the proof, I only revised it to use the `F/_ x C` explicit "not free in" hypothesis instead of the distinct variable statement, and added the **`f`** suffix. Therefore I kept Mario's attribution and only added my name as "Revised by", even though I originally only included the new theorem in my mathbox. That's why it's a theorem authored by Mario, but pulled from my Mathbox.